### ISSUE TITLE
Install glibc dependancy error

### ISSUE THREAD

**Issue by bterps** [jay0lee/got-your-back#331]
The issue tracker is for reporting product deficiencies. How do I questions should be posted to the discussion forum at https://groups.google.com/group/got-your-back. When in doubt, start at the discussion forum and return here only when instructed to do so.

Please confirm the following:
* I have upgraded to the latest GYB release from https://github.com/jay0lee/got-your-back/releases and I still have this issue.
* I am typing the command as described in the GAM Wiki at https://github.com/jay0lee/got-your-back/wiki

Full steps to reproduce the issue:
1. run bash <(curl -s -S -L https://git.io/gyb-install)
2.
3. Profit

Expected outcome (what are you trying to do?):
Install GYB
Actual outcome (what errors or bad behavior do you see instead?):
```
This Linux distribution uses glibc 2.24
Using GYB compiled against glibc2.23

Checking GitHub URL https://api.github.com/repos/jay0lee/got-your-back/releases for latest GYB release (unauthenticated)...

Getting file and download URL...

ERROR: Attribute: browser_download_url for linux-x86_64-glibc2.23.tar.xz version latest not found
```
Someone else on the message board had a similar issue back in May

**Comment by jay0lee** [comment]
Can you try this now? It should be fixed with 1.53.

**Comment by bterps** [comment]
can confirm, thank you!!
